src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/partially_quiet.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Test DeprecationErrorHandler quiet on everything bug self deprecations with no
+self deprecations on vendor deprecation
+--FILE--
+<?php
+
+$k = 'SYMFONY_DEPRECATIONS_HELPER';
+putenv($k.'='.$_SERVER[$k] = $_ENV[$k] = 'max[self]=0&verbose[self]=1');
+putenv('ANSICON');
+putenv('ConEmuANSI');
+putenv('TERM');
+
+$vendor = __DIR__;
+while (!file_exists($vendor.'/vendor')) {
+    $vendor = dirname($vendor);
+}
+define('PHPUNIT_COMPOSER_INSTALL', $vendor.'/vendor/autoload.php');
+require PHPUNIT_COMPOSER_INSTALL;
+require_once __DIR__.'/../../bootstrap.php';
+require __DIR__.'/fake_vendor/autoload.php';
+require __DIR__.'/fake_vendor/acme/lib/deprecation_riddled.php';
+
+?>
+--EXPECTF--
+Unsilenced deprecation notices (3)
```

Comment:
I don't see self deprecations here. Showing that self deprecations are non-verbose was the goal of this test case right?

Reply:
No, it's everything but self deprecation is verbose => unsilenced is not verbose

Reply:
I'm confused, this

> self deprecations are non-verbose

and

> everything but self deprecation is verbose

reads same to me 🤔 


Anyway I just targetted first line of output, I was trying to point out self deprecation section is missing in output

Reply:
I'll add better tests later, this one is confusing (there are no self deprecations, some would be nice)